src/main/java/com/ericsson/eiffel/remrem/shared/MsgService.java

```diff
@@ -5,4 +5,6 @@
 
 public interface MsgService {
     String generateMsg(String msgType, JsonObject bodyJson);
+    String validateMsg(String msgType, JsonObject bodyJson);
```

Comment:
I think we will add this code to the new repository that is going to be created for this purpose